src/lib/updatableList.ts

```diff
@@ -8,13 +8,25 @@ import List from 'list.js';
  * @param {object[]} values values to add to the list
  * @returns {List} List.js List
  */
-function createList(element, options, values) {
+function createList(
+  element: string | HTMLElement,
+  options?: List.ListOptions,
+  values?: object[]
+) {
```

Comment:
```suggestion
): List {
```

This is perhaps my preference, though.